src/test/resources/test.template.properties

```diff
@@ -43,7 +43,7 @@ test.timeout=15
 # Set to true to enable GodMode.
 # Please read the GodMode documentation if you are not yet familiar with it, and use with care.
 # Remember to set back to false when done and rerun the test(s).
-test.godmode.enabled=false
```

Comment:
You don't push this file. To turn on `God Mode`, change `test.properties` instead.

Reply:
@bqnguyen94 I was thinking of doing the changes and rebase before asking for review.